lib/nxt_pipeline/constructor.rb

```diff
@@ -0,0 +1,13 @@
+module NxtPipeline
+  class Constructor
+    def initialize(name, opts, block)
+      @name = name
+      @block = block
+      @opts = opts
+    end
+
+    attr_reader :block, :opts
+
+    delegate_missing_to :block
```

Comment:
We should add a `>= 5.1` version constraint for the active support dependency in the `.gemspec` file, because `delegate_missing_to` is not available in 5.0. Might also make sense to bump to `0.3` then.

Reply:
👍

Reply:
Probably could also use simple delegator to be backwards compatible.

Reply:
Or even better just implement call :-)

Reply:
Will bump version and release once reviewed.